tools/distpartitioning/data_shuffle.py

```diff
@@ -155,9 +161,7 @@ def exchange_edge_data(rank, world_size, edge_data):
     edge_data.pop(constants.OWNER_PROCESS)
     return edge_data
 
-
-def exchange_node_features(rank, world_size, node_data, node_features, ntypes_map, \
-        ntypes_nid_map, ntype_id_count, node_part_ids):
+def exchange_node_features(rank, world_size, node_feature_tids, ntype_gnid_map, node_part_ids, node_features):
```

Comment:
you changed the input arguments. you need to update the doc string accordingly.

Reply:
Updated the doc-string to match with the updated inpur arguments to this function as suggested.